### PR TITLE
Created my profile.

### PR DESCRIPTION
**Pull request by OmAvhad**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1636"></a>

### CODE REVIEW

**Review by SaptarshiSarkar12 — APPROVED** [review]
LGTM 👍 . The changes are ready to be merged.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant, Reusable, Maintainable

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

File: public/data/OmAvhad.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+  "name": "Om Avhad",
+  "type": "personal",
+  "bio": "Backend Developer",
+  "avatar": "https://avatars.githubusercontent.com/u/89376666?v=4",
```
Comment (Pradumnasaraf) [inline public/data/OmAvhad.json:5]:
```suggestion
  "avatar": "https://github.com/OmAvhad.png",
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Using the `https://github.com/OmAvhad.png` will me more dynamic as you change your profile picture in the future.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

**Comment by SaptarshiSarkar12** [comment]
@Panquesito7 @kumarsonsoff3 Please check if this PR is ready to be merged.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good to me! 👍 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Comment by OmAvhad** [comment]
Hello,

Can some one help me with this, Format test is failing !
@SaptarshiSarkar12 @Pradumnasaraf

**Comment by kumarsonsoff3** [comment]
No worries @OmAvhad, this issue is resolved last day! This PR is good to goo! 

It will re-run when we'll push your PR!! 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**